src/backend/distributed/shardsplit/shardsplit_decoder.c

```diff
@@ -10,10 +10,14 @@
 #include "postgres.h"
```

Comment:
should we rename this file? Or separate it into 2 pieces?

Reply:
OK. I have split the CDC related changes to a seperate file cdc_decoder.c